files/clean.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
 python3 clean.py
 rm -rf files/
+rm -rf gentoo/
```

Comment:
LGTM :+1:

Reply:
Whilst I have considered nuking the Gentoo repo clone, it's unnecessary, as the repo clone is atomically correct (checking Buildbot docs on this confirms).
I've managed to solve the Manifest issue by another patch, which I will upload shortly in a PR.